merlion/models/forecast/base.py

```diff
@@ -176,7 +176,7 @@ def train(self, train_data: TimeSeries, train_config=None) -> Tuple[TimeSeries,
         return super().train(train_data=train_data, train_config=train_config)
 
     def train_post_process(
```

Comment:
For forecasters, `train_post_process()` will apply the inverse transform to the train pred/stderr if `invert_transform` is True. For pure anomaly detectors, it will train the post rule. For forecasting-based anomaly detectors, it will first apply the inverse transform to the train pred/stderr (if desired), convert pred/stderr to anomaly scores, and then train the post rule.